packages/hint-content-type/tests/tests.ts

```diff
@@ -10,7 +10,7 @@ import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
 const hintPath = getHintPath(__filename);
 
 const pngImage = fs.readFileSync(`${__dirname}/fixtures/image.png`); // eslint-disable-line no-sync
-const svgImage = '<svg xmlns="http://www.w3.org/2000/svg"><path d="M1,1"/></svg>';
+const svgImage = '<svg xmlns="http://www.w3.org/2000/svg" width="1px" height="1px"><path d="M1,1"/></svg>';
```

Comment:
Given you opened #2204 later which also includes this change, I'm assuming you intend to rebase this PR once that one is merged, correct?

Reply:
Yes, this PR started failing in CI (on my mac it runs pretty well) and I wanted to get this change sooner rather than later.